tests/static_lib/build.rs

```diff
@@ -0,0 +1,9 @@
+fn main() {
+    windows::build! {
+        Microsoft::Web::WebView2::Win32::*,
+        Windows::Win32::Storage::FileSystem::{
+            GetFileVersionInfoSizeW, GetFileVersionInfoW, VerQueryValueW,
+        },
+        Windows::Win32::UI::Shell::GetCurrentProcessExplicitAppUserModelID,
```

Comment:
These don't seem to be needed for the test.

Reply:
The link fails with just `Microsoft::Web::WebView2::Win32::*` because the static library has these missing imports. It probably only needs 1 from each DLL/import lib to add it to the linker command line, but I figured it was better to be specific about which ones we're pulling in.

I guess the `build.rs` script could also just add them to the link with a cargo directive on `stdout`. That would have the advantage of building a smaller bindings crate.